src/modules/tests/test-status/test-status.model.ts

```diff
@@ -0,0 +1,5 @@
+export enum TestStatus {
+  NotStarted = 'NotStarted',
```

Comment:
I was wondering if the status 'booked' would be more appropriate (instead of 'NotStarted') to keep it more in line with the model on wiki - if it's on DE's journal it means that it's scheduled/booked

Reply:
Good shout, I will use 'Booked' instead.